neo4j-cli/aura/internal/subcommands/import/job/cancel.go

```diff
@@ -0,0 +1,59 @@
+package job
+
+import (
+	"fmt"
+	"github.com/neo4j/cli/common/clicfg"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/api"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/output"
+	"github.com/spf13/cobra"
+	"log"
+	"net/http"
+)
+
+func NewCancelCommand(cfg *clicfg.Config) *cobra.Command {
+	var (
+		projectId string
+		jobId     string
+	)
+
+	const (
+		projectIdFlag = "project-id"
+		jobIdFlag     = "job-id"
+	)
+
+	cmd := &cobra.Command{
+		Use:   "cancel <id>",
+		Short: "Cancel a job by id",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if jobId == "" {
+				return fmt.Errorf("jobId is required")
+			}
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path := fmt.Sprintf("/projects/%s/import/jobs/%s/cancel", projectId, jobId)
```

Comment:
This is for the Aura API endpoint rather than the code.  

Why are you doing a PATCH to /projects/{projectId}import/jobs/{jobId}/cancel rather than a DELETE to /projects/{projectId}import/jobs/{jobId} ?

Reply:
@LackOfMorals - when we cancel a job, the job itself is still persisting in our side. So I'm wondering if DELETE is fit in this scenario? Additionally, in the future, we might introduce `DELETE /jobs/:id` for user to delete the persisted job history.

Reply:
You can still do a DELETE in that scenario and you would respond with a 202 status code.  that would indicate that a request has been accepted for processing, but processing has not been completed.   This is exactly what happens when you delete an aura instance - it takes time to dismantle everything. 

Unless you mean cancel does not remove the job? It just stops the job and leaves it still there for a future update?